front/app/containers/CustomPageShow/CustomPageEvents.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import EventsWidget from 'components/LandingPages/citizen/EventsWidget';
+import { isNilOrError } from 'utils/helperUtils';
+import useAdminPublications from 'hooks/useAdminPublications';
+import { ICustomPageData } from 'services/customPages';
+
+interface Props {
+  page: ICustomPageData;
+}
+
+const CustomPageEvents = ({ page }: Props) => {
+  // There will be either topic or area ids if this component renders.
+  // To enable it, the page needs either a topic or area associated with it.
+  const topicIds = page.relationships.topics.data.map((topic) => topic.id);
+  const areaIds = page.relationships.areas.data.map((area) => area.id);
+
+  const adminPublications = useAdminPublications({
+    topicFilter: topicIds,
+    areaFilter: areaIds,
+    publicationStatusFilter: ['published', 'archived'],
+  });
+  const projectIds = !isNilOrError(adminPublications.list)
```

Comment:
It seems I'll need the same array in my `ProjectsList` component. So, maybe it makes sense to create it in `front/app/containers/CustomPageShow/index.tsx` so I could reuse it?

Reply:
@alexander-cit Or we need to make this component `CustomPageProjectsAndEvents` in your branch, because we don't need it if projects are turned off?

Reply:
Yes, sounds good. I'll do it.